src/util/system.cpp

```diff
@@ -415,7 +421,6 @@ const fs::path& ArgsManager::GetBlocksDirPath() const
     path /= fs::PathFromString(BaseParams().DataDir());
     path /= "blocks";
     fs::create_directories(path);
-    path = StripRedundantLastElementsOfPath(path);
```

Comment:
In commit "refactor: replace boost::filesystem with std::filesystem" (41d7166c8a598b604aad6c6b1d88ad46e23be247)

Note: Just to explain this change, this line is deleted because the `path` variable is set 2 lines above with literal "blocks" last element, so this line would never do anything, even before this PR. Earlier discussion was https://github.com/bitcoin/bitcoin/pull/20744#discussion_r731366070